azkaban-common/src/test/java/azkaban/executor/ExecutorManagerTest.java

```diff
@@ -162,7 +170,8 @@ public void testSetupExecutorsException() throws ExecutorManagerException {
     Executor executor1 = loader.addExecutor("localhost", 12345);
 
     ExecutorManager manager =
-      new ExecutorManager(props, loader, new AlerterHolder(props));
+      new ExecutorManager(props, loader, new AlerterHolder(props),
```

Comment:
This is an example of why Guice is helpful. Otherwise, we may have more similar code in the production code.